test/libsolidity/syntaxTests/inlineAssembly/shadowing/global_function_by_opcode.sol

```diff
@@ -0,0 +1,8 @@
+function mload() pure {}
+contract C {
+    function g() public pure {
+        assembly {
+        }
+    }
+}
+// ----
```

Comment:
Was this test producing a warning before because `nameFromCurrentScope` returned the user defined function declaration named `mload`?

Reply:
It was producing a warning that `mload` and the `mload` instruction are in conflict.